CONTRIBUTING.md

```diff
@@ -146,9 +146,9 @@ of binaries in ./build directory.
 
 In order to test locally, you'd need a test setup. Local test setup automation is planned but but not done yet, so currently the environment has to be set up manually.
 
```

Comment:
Please add a subheader for "Local Test bed setuop" , currently it's mixed up with other stuff